keyvault/client/secrets/README.md

```diff
@@ -0,0 +1,252 @@
+# Azure Key Vault Secret client library for Java
+Azure Key Vault is a tool for securely storing and accessing secrets. A secret is anything that you want to tightly control access to, such as API keys or passwords. A vault is logical group of secrets.
```

Comment:
I see this is coming from https://docs.microsoft.com/en-us/azure/key-vault/key-vault-whatis#basic-concepts ... 
I am wondering if in the intro we should have something more generic about what Key Vault actually is. i.e. include that it stores more than secrets, so that we can use the same 2 description phrase in all docuemnts for all clients.
For example something like here: 
" Cloud applications and services use cryptographic keys and secrets to help keep information secure. Azure Key Vault safeguards these keys and secrets.
The Secret client library allows you to securely store and tightly.... [everything else you have] "
From https://docs.microsoft.com/en-us/azure/key-vault/key-vault-whatis#basic-concepts

Reply:
I have [this[(https://github.com/samvaity/azure-sdk-for-python/tree/secrets-track2-readme/sdk/keyvault/azure-security-keyvault/azure/security/keyvault/secrets#azure-key-vault-secret-client-library-for-python) in the Secrets README
And [this](https://github.com/samvaity/azure-sdk-for-python/tree/keyvault-track2-readme/sdk/keyvault/azure-security-keyvault#azure-key-vault-client-library-for-python) in the keyvault README which would be more generic to Key Vault and not Keyvault in context of secrets.

@maririos ^^